src/Test.UnitTests.Sarif/Test.UnitTests.Sarif.csproj

```diff
@@ -50,6 +52,11 @@
     <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\OneRunWithAllReportingDescriptors.sarif" />
   </ItemGroup>
 
+  <ItemGroup>
+    <EmbeddedResource Include="TestData\PrereleaseCompatibilityTransformer\ExpectedOutputs\ThreadFlowLocationWithKind.sarif" />
```

Comment:
>TestData [](start = 31, length = 8)

Did VS add these lines? I'm just surprised it didn't add them in the `ItemGroup` below starting on Line 71.

Reply:
thanks for noticing this, i will cross check the way i set the properties of these new files

---
In reply to: [288690305](https://github.com/microsoft/sarif-sdk/pull/1510#discussion_r288690305) [](ancestors = 288690305)

Reply:
i manually edited the file to show all embedded resources in a single group.

---
In reply to: [288705602](https://github.com/microsoft/sarif-sdk/pull/1510#discussion_r288705602) [](ancestors = 288705602,288690305)